addon/components/polaris-breadcrumbs.hbs

```diff
@@ -0,0 +1,41 @@
+<nav
+  role="navigation"
+  data-test-breadcrumbs
+  ...attributes
+>
+  {{#if this.breadcrumb.url}}
+    <PolarisUnstyledLink
+      class="Polaris-Breadcrumbs__Breadcrumb"
+      @url={{this.breadcrumb.url}}
+      @ariaLabel={{this.breadcrumb.accessibilityLabel}}
+      @onMouseUp={{this.handleMouseUpByBlurring}}
+    >
+      <span class="Polaris-Breadcrumbs__ContentWrapper">
+        <span data-test-breadcrumb-icon class="Polaris-Breadcrumbs__Icon">
+          {{!-- TODO #polaris-v5-newDesignLanguage add newDesignLanguage icon, styles, and visually-hidden wrapper --}}
```

Comment:
@tomnez I've merged #678 which adds support for this new design language option.
Let's update this and remove the TODO before considering this done